index.js

```diff
@@ -58,82 +61,83 @@ function Jasmine2ScreenShotReporter(opts) {
+        '<%= mark %>' +
+        '<%= name %> ' +
+        '(<%= duration %> s)' +
+        '<%= reason %>' +
         '</li>'
     );
 
     var openReportTemplate = _.template(
         '<html>' +
-            '<head>' +
-                '<meta charset="utf-8">' +
-                '<style>' +
-                    'body { font-family: Arial; }' +
-                    'ul { list-style-position: inside; }' +
-                    'span.passed { padding: 0 1em; color: green; }' +
-                    'span.failed { padding: 0 1em; color: red; }' +
-                    'span.pending { padding: 0 1em; color: orange; }' +
-                '</style>' +
-                '<%= userCss %>' +
-                '<script type="text/javascript">' +
-                  'function showhide(id) {' +
-                    'var e = document.getElementById(id);' +
-                    'e.style.display = (e.style.display == "block") ? "none" : "block";' +
-                  '}' +
-                  'function buildQuickLinks() {' +
-                    'var failedSpecs = document.querySelectorAll("li.failed");' +
-                    'var quickLinksContainer = document.getElementById("quickLinks");' +
-                    'for (var i = 0; i < failedSpecs.length; ++i) {' +
-                      'var li = document.createElement("li");' +
-                      'var a = document.createElement("a");' +
-                      'a.href = "#" + failedSpecs[i].id;' +
-                      'a.textContent = failedSpecs[i].dataset.name + "  (" + failedSpecs[i].dataset.browser + ")";' +
-                      'li.appendChild(a);' +
-                      'quickLinksContainer.appendChild(li);' +
-                    '}' +
-                  '}' +
-                  'function updatePassCount() {' +
-                    'var totalPassed = document.querySelectorAll("li.passed").length;' +
-                    'var totalFailed = document.querySelectorAll("li.failed").length;' +
-                    'var totalSpecs = totalFailed + totalPassed;' +
-                    'console.log("passed: %s, failed: %s, total: %s", totalPassed, totalFailed, totalSpecs);' +
-                    'document.getElementById("summaryTotalSpecs").textContent = ' +
-                    'document.getElementById("summaryTotalSpecs").textContent + totalSpecs;' +
-                    'document.getElementById("summaryTotalFailed").textContent = ' +
-                    'document.getElementById("summaryTotalFailed").textContent + totalFailed;' +
-                    'if (totalFailed) {' +
-                      'document.getElementById("summary").className = "failed";' +
-                    '}' +
-                  '}' +
-                  'function start() {' +
-                    'updatePassCount();' +
-                    'buildQuickLinks();' +
-                  '}' +
-                  'window.onload = start;' +
-                '</script>' +
-            '</head>' +
-            '<body>'
+        '<head>' +
```

Comment:
Alright, well I guess I can figure that out after the merge and before releasing new version.